cabal.project

```diff
@@ -13,3 +13,8 @@ package ghcide
 write-ghc-environment-files: never
 
 index-state: 2020-07-01T21:29:04Z
+
+allow-newer:
+  floskell:aeson
+  HsYAML-aeson:aeson
+  stylish-haskell:aeson
```

Comment:
We probably want to remove these before merging.

I've got an eye on the relevant PRs - thanks @bubba for opening them where necessary.

Reply:
What are the remaining packages blocked by aeson-1.5?

Reply:
Floskell got the bump, but there hasn't been a Hackage release with it.

We're still waiting on Stylish.